### ISSUE TITLE
Deafening also mutes sounds

### ISSUE THREAD

**Issue by ghost** [mumble-voip/mumble#1293]
Deafening yourself also makes sound effects unhearable. I understand this may be intended behavior. It may perhaps be a good idea to add a checkbox in the settings toggling whether you want this or not. This would come in very handy when using Mumble as a chat client where you're both muted and deafened but not AFK to have an audible hint when a chat message is received.

**Comment by ZeroAbility** [comment]
Has this been implemented? 1.4 has checkboxes to disable the soundfile for events which seems fine grained as opposed to an all-or-nothing approach to disabling Mumble notification playback. Similar to this request:

https://github.com/mumble-voip/mumble/issues/1257

... there are simpler methods to disable audio notifications per application built in to the operating systems we support.

**Comment by Hartmnt** [comment]
> Has this been implemented? 1.4 has checkboxes to disable the soundfile for events which seems fine grained as opposed to an all-or-nothing approach to disabling Mumble notification playback.

No, if you deafen yourself, you will not hear any notification sounds. This is not related to the checkboxes you can set.

> Similar to this request:

Actually, this issue is more related to #2997
If you can override not hearing sounds while being deafened for certain notification types, that would be one part of what is necessary for #2997
